sycl/include/sycl/ext/intel/experimental/fpga_lsu.hpp

```diff
@@ -51,8 +51,10 @@ template <class... _mem_access_params> class lsu final {
 public:
   lsu() = delete;
 
-  template <typename _T, access::address_space _space, typename _propertiesT>
-  static _T load(sycl::multi_ptr<_T, _space> Ptr, _propertiesT Properties) {
+  template <typename _T, access::address_space _space,
+            access::decorated _is_decorated, typename _propertiesT>
```

Comment:
```suggestion
            access::decorated _Is_decorated, typename _propertiesT>
```

Please note that if you want to use a reserved identifier, it should either start with double underscore, or a single underscore followed by an upper case letter: [cppreference](https://en.cppreference.com/w/cpp/language/identifiers).

Not asking to fix existing `_lower_case` occurrences in the file, but at least we can attempt not to introduce more of them